core/app/models/spree/customer_return.rb

```diff
@@ -1,5 +1,6 @@
 module Spree
   class CustomerReturn < Spree::Base
+    include Spree::Core::NumberGenerator.new(prefix: 'CR', length: 9)
```

Comment:
Ohh, I was not aware that users of the spree engine should have the choice to configure this explicitly.

The prior configuration mechanism required to do a "code change", not a configuration change. I consider overwriting a method a code change, where assigning a literal to a global object a configuration change.

I could easily change this PR so that each model exposes a mutable Hash users could change the configuration of that particular number generator with.

@JDutil Does this fit your intention?